hera_cal/abscal.py

```diff
@@ -998,6 +998,8 @@ def abscal_arg_parser():
     a.add_argument("--bl_taper_fwhm", default=None, type=float, help="enact gaussian weight tapering based on baseline length [meters] with specified FWHM.")
     a.add_argument("--window", default=None, type=str, help="window to enact on data before FFT in delay solvers, options=[None, 'blackmanharris', 'hann']")
     a.add_argument("--edge_cut", default=0, type=int, help="number of channels to flag on each band-edge in delay and global phase solvers.")
+    a.add_argument("--solar_flag", default=False, action='store_true', help="FLag data when the sun is above some specified altitude.")
```

Comment:
Can we make this a single parameter. If you want to make the default to never flag, regardless of where the sun is, perhaps just make the "solar_flag_altitude" default to 90 degrees.

Reply:
Sure thing, i'm happy to do that. in that case it would be default of -90 degrees though..

Reply:
If you only flag data *above* some altitude and you want to never flag by default, then the default altitude should be 90, since the sun is never above 90.